http-testing/src/test/resources/parameterized_csv/hello_world_test_parameterized_csv_source_files.json

```diff
@@ -0,0 +1,25 @@
+{
+    "scenarioName": "Fetch and assert GitHub userIds by their userNames",
+    "steps": [
+        {
+            "name": "get_user_details",
+            "url": "/users/${0}",
+            "method": "GET",
+            "request": {
+            },
+            "assertions": {
+                "status": 200,
+                "body": {
+                    "login" : "${0}",
+                    "type" : "User",
+                    "name" : "${1}",
+                    "location" : "${2}",
+                    "id" : "$EQ.${3}"
+                }
+            }
+        }
+    ],
+    "parameterized": {
+        "csvSource":"parameterized_csv/params.csv"
```

Comment:
Updated code to read from resources folder without referring to "./test/...".
Changed method impl to include Path prefix to "./test/...".

There shohuld be a better way to do this, I have put this as a quick fix. Please feel free to amend this.

Reply:
@imprashant , any news on this comment ? Or are you happy for us to merge it?